docs/admin/admin-ui-setup/templates.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Templates
+summary: This article describes the templates form. This form allows explains what a template is, and where it is used..
+authors:
+    - Jason Novich
+date: 2023-oct-22
+---
+
+A template is a pre-set configuration that is used help quickly configure assets in the system. Using a template allow you to submit a workload based on template which has aggregated all relevant building blocks for your workload.
+
+## Creating Templates
+
+To create a template:
+
+1. In the left menu, press *Templates*, then press *New Template*.
+2. In the *Scope* pane, select a *Scope*.
+3. In the *Template name* pane, enter a name.
+4. In the *Environment* pane, select an environment from the list. Use the *Search environments* if you do not see your environment listed. Press *New environment* to add a new environment to the system. Press *More settings* to add an `Environment variable` or to edit the *Command* and *Arguments* field for the environment you selected.
```

Comment:
we need to mention here that the environment is the only real mandatory asset to select in a template. In other words- the template is not obliged to state the data source or compute resource, but the environment is a must

Reply:
@liohill the template comes with a "default" environment variable. 
![image](https://github.com/run-ai/docs/assets/124490127/fc1d0e0d-53f3-434d-b9b8-1b741b85e0ca)

Reply:
> we need to mention here that the environment is the only real mandatory asset to select in a template. In other words- the template is not obliged to state the data source or compute resource, but the environment is a must

@liohill The envorinment variable comes as default when you create the environment. See image:
![image](https://github.com/run-ai/docs/assets/124490127/4ee1c8c0-d8cf-4e5c-af9e-824de59d4852)
So even if you do not choose to make changes to this, the field is there. You can add more than one env variable.

Maybe this:
```suggestion
4. In the *Environment* pane, select an environment from the list. Use the *Search environments* if you do not see your environment listed. Press *New environment* to add a new environment to the system. Press *More settings* to edit or add an `Environment variable`, or to edit the *Command* and *Arguments* field for the environment you selected.
```